cdb2jdbc/src/main/java/com/bloomberg/comdb2/jdbc/DatabaseDiscovery.java

```diff
@@ -44,13 +45,12 @@ public class DatabaseDiscovery {
     static final String CDB2DBCONFIG_SECONDARY_PATH = "/bb/etc/cdb2/config.d/";
 
     /* comdb2lcldb, per jvm. */
-    static class TimeAndHosts {
+    private static final class TimeAndHosts {
```

Comment:
`TimeAndHosts` isn't used anywhere except this file, so it might as well be private.